Document/0x06b-Basic-Security-Testing.md

```diff
@@ -516,3 +516,15 @@ PID  Name
 ##### Troubleshooting.
 
 If something goes wrong (which it usually does), mismatches between the provisioning profile and code signing header are the most likely suspect. In that case it is helpful to read the [official documentation](https://developer.apple.com/library/contehttps://support.portswigger.net/customer/portal/articles/1841108-configuring-an-ios-device-to-work-with-burpnt/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingProfiles/MaintainingProfiles.html "Maintaining Provisioning Profiles") and gaining a deeper understanding of the code signing process. Apple's [entitlement troubleshooting page](https://developer.apple.com/library/content/technotes/tn2415/_index.html "Entitlements Troubleshooting ") is also a useful resource.
+
+##### Network Monitoring/Sniffing on Non-Jailbroken Devices
```

Comment:
Are you sure this is the correct title? The content below seems to be about sniffing non-HTTP traffic. What is non-jailbroken specific about this?